applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -151,11 +155,8 @@ static void pa_sync_lost_cb(struct bt_audio_broadcast_sink *sink)
 
 	LOG_INF("Sink disconnected");
 
-	/* At this point the broadcast sink is stopped and deleted */
-	synced_to_broadcast = false;
-	broadcast_sink = NULL;
-
-	broadcast_sink_cleanup();
+	ret = bis_headset_cleanup(true);
+	ERR_CHK_MSG(ret, "Error cleaning up");
 
 	LOG_INF("Restarting scanning for broadcast sources");
 
```

Comment:
We might need to use BT_LE_SCAN_PASSIVE in the next line for keeping the consistency.

Reply:
Nice catch, fixed this now